pkg/wasm/cache.go

```diff
@@ -175,18 +175,15 @@ func urlAsResourceName(fullURLStr string) string {
 }
 
 func pullIfNotPresent(pullPolicy extensions.PullPolicy, u *url.URL) bool {
-	if u.Scheme == "oci" {
-		switch pullPolicy {
-		case extensions.PullPolicy_Always:
-			return false
-		case extensions.PullPolicy_IfNotPresent:
-			return true
-		default:
-			return !strings.HasSuffix(u.Path, ":latest")
-		}
+	switch pullPolicy {
```

Comment:
rename this function to `ignoreResourceVersion` to match the real meaning, WDYT?

Reply:
Hm, it makes sense. Will change it.

Reply:
Done